analysis_driver/reader/demultiplexing_parsers.py

```diff
@@ -326,5 +326,37 @@ def parse_welldup_file(welldup_file):
                 in_summary = False
     return dup_per_lane
 
-
+def parse_adapter_trim_file(adapter_trim_file, run_id):
+    adapters_trimmed_by_id = {}
+    with open(adapter_trim_file) as open_file:
+        open_file = open_file.read()
+        open_file = (open_file.split('\n\n')[0]).split('\n')
+        for line in islice(open_file, 1, None):
+            lane = line.split()[0]
+            read = line.split()[1]
+            sample_id = line.split()[3]
+            trimmed_bases = line.split()[6]
+            id = (run_id, sample_id, lane)
```

Comment:
We should avoid using the keyword ```id``` ad it is also a builtin function in python
see https://docs.python.org/3.4/library/functions.html#id